money-monkey-web/app/ui/PopupModal.tsx

```
@@ -0,0 +1,65 @@
+  return (
+    <div
+      className={`transition w-screen h-screen fixed left-0 top-0 flex items-center justify-center ${
+        visible ? "bg-gray-200 bg-opacity-50" : "pointer-events-none"
+      }`}
+      onClick={onClose}
+      onKeyDown={(e) => {
+        if (e.key === "Escape") {
+          onClose(e);
+        }
+      }}
+      tabIndex={0}
```

Biome

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)